.github/workflows/main.yml

```diff
@@ -25,3 +25,36 @@ jobs:
       - uses: actions/checkout@v1
       - uses: './'
         env: ${{matrix.env}}
+  industrial_ci_external:
```

Comment:
It  might make sense to push this into a second workflow that is only run for PRs

Reply:
Done in 06312bc45cf78bc2269dae35d1494e34178aa140

Reply:
I might not end-up making sense
- **ABI_CHECK** was checked using external repository
- **Github Action** is so fast that all jobs run almost concurrently and finish within 12min

Reply:
> might not end-up making sense

Then we can run both for PRs and pushes

Reply:
The number of minutes might be limited per organization, so we might want to save some.
@gavanderhoorn: Do you know more about it?

Reply:
It is 20 concurrent job per pax for free account, 60 for team account at the moment. ([reference](https://docs.github.com/en/free-pro-team@latest/actions/reference/usage-limits-billing-and-administration#usage-limits))

Reply:
> ABI_CHECK was checked using external repository

moved these to main